### ISSUE TITLE
Test harness to validate YAML configuration extension

### ISSUE THREAD

**Issue by guidograzioli** [ansible-middleware/wildfly#72]
##### SUMMARY

Some unit testing is needed on the configuration passed via the collection to the YAML configuration feature of wildfly >= 16.1
The test harness should setup configuration points in YAML, and then check the settings were properly applied after starting the service up, eventually in a `wildfly_validation` role.

A list of "good to test" configuration points are:

* **standard-sockets**
```
socket-binding-group:
    standard-sockets:
        remote-destination-outbound-socket-binding:
            [...]
```
* **modcluster**
```
  subsystem:
    modcluster:
      proxy:
        default:
```
* **messagging**
```
 subsystem:
    messaging-activemq:
      server: [...]
      remote-connector: [...]
      pooled-connection-factory: [...]
```
* **ejb/ee**
```
    ejb3:
      default-resource-adapter-name: RemoteConnectionFactory
    ee:
      service:
        default-bindings:
          jms-connection-factory: "java:jboss/RemoteConnectionFactory"
```

When supported (not yet):

* system properties
* jgroups stacks (ie: [this cli script](https://github.com/ansible-middleware/aws-eapamq-demo/blob/main/files/jgroups-jboss-cli.j2))
* queues/topics (ie: [this](https://github.com/ansible-middleware/aws-eapamq-demo/blob/main/files/messaging-jboss-cli.j2))

##### ISSUE TYPE
 - Feature Idea

**Comment by rpelisse** [comment]
@RanabirChakraborty here is the issue we discussed! Feel free to ask question here is you need help to understand the requirements!

**Comment by RanabirChakraborty** [comment]
Thanks, @guidograzioli , I have started working on this issue. But If possible can you please provide me with some examples? so that I can move forward with it.

**Comment by rpelisse** [comment]
@RanabirChakraborty you just defintely added a test inspired by this use case [WFCORE-5918](https://issues.redhat.com/browse/WFCORE-5918)

**Comment by rpelisse** [comment]
Fixed by #87
